### DISCUSSION TITLE
vim_lsp_inlayhint error

### DISCUSSION THREAD

**Discussion by joaoLouceiro** [nvim-java/nvim-java#312] [General]
Hey, vim experts, can someone give me a hand, here?
Every time I add a new line to a java file, I get this error:
![image](https://github.com/user-attachments/assets/e8f05566-7696-4c8e-89b8-7ef776ee780c)

Everything seems to run fine, except for this annoying error.
Any help?

**Comment by s1n7ax** [comment]
I think people were seen this in neovim 0.10. This plugin does not do anything related to inlay hint but rather neovim inlayhint directly talks to language server. I guess the solution would be to move to 0.11.

**Reply by joaoLouceiro** [reply]
Hey, @s1n7ax , thanks for the answer!

Tried updating but everything is the same. The "solution" was to setup lspconfig's  inlay_hints to exclude java files.

Best regards and keep up the good work!
